packages/origin-247-certificate/package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "@energyweb/origin-247-certificate",
+    "version": "0.0.1",
+    "main": "./dist/js/src/index.js",
+    "types": "./dist/js/src/index.d.ts",
+    "scripts": {
+        "test:unit": "jest ",
+        "test:e2e": "concurrently --success first --kill-others -n eth,test \"yarn run:ganache\" \"wait-on tcp:8545 && yarn drop && yarn migrate && jest --testRegex '.e2e-spec.ts$'\"",
+        "build": "tsc -b tsconfig.json",
+        "migrate": "yarn typeorm:run:issuer",
+        "drop": "yarn typeorm:drop && yarn typeorm:drop:issuer",
+        "typeorm:drop:issuer": "node_modules/typeorm/cli.js schema:drop --config node_modules/@energyweb/issuer-api/dist/js/ormconfig.js",
+        "typeorm:drop": "yarn typeorm:drop:issuer",
+        "typeorm:run:issuer": "node_modules/typeorm/cli.js migration:run --config node_modules/@energyweb/issuer-api/dist/js/ormconfig.js",
+        "run:ganache": "ganache-cli -q -m 'govern long helmet alert stay supply kick knife boss until buzz unlock' -l 8000000 -e 1000000 -a 20"
```

Comment:
We should generate a new mnemonic to be used for this SDK, but let's do this in some other PR.

Reply:
It's for the development only

Reply:
The instructions are not precise though - because it's wrapper for issuer-api, then it requires one to execute typeorm migration on ones application